sdk/tables/Azure.Data.Tables/tests/TableClientLiveTests.cs

```diff
@@ -27,6 +27,31 @@ public class TableClientLiveTests : TableServiceLiveTestsBase
             endpointType /* To record tests, add this argument, RecordedTestMode.Record */)
         { }
 
+        [RecordedTest]
+        public async Task UpsertAndQueryWithSingleQuoteNames([Values(true, false)] bool expressionQuery)
+        {
+            List<TableEntity> entityResults;
+            string partitionKeyValue = "PartitionWith'singleQuote";
```

Comment:
What about a double quote?

Reply:
double quote or 2 single quotes? The spec only mentions single quotes - https://docs.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#single-quote-

Reply:
Two single quotes.

Reply:
Updated the test with two quotes just to verify.